USER-INTERFACE-INTERFACE.md

```diff
@@ -640,41 +670,73 @@ If the UI is remembering the database password, it should forget it when this me
+derivation paths is supplied. If no derivation paths are supplied, this parameter is ignored.
+
+`mnemonicPhrase` is the mnemonic phrase that represents the seed. It must have 12, 15, 18, 21, or 24 words.
+
+`mnemonicPhraseLanguage` is the language in which the mnemonic phrase is supplied. Acceptable values are
+"English", "Chinese", "Traditional Chinese", "French", "Italian", "Japanese", "Korean", and "Spanish".
 
 `mnemonicPassphraseOpt`, if specified, is the "25th word" in the mnemonic passphrase: that is, an additional word
 (it can be any word; it's not constrained to the official mnemonic-phrase list) that was used along with the
 words of the mnemonic phrase to generate the seed number from which the consuming and possibly earning wallets
-were derived. If no mnemonic passphrase was used to generate the wallets, this value must be absent.
+were derived. If no mnemonic passphrase was used to generate the wallets, this value must be null or absent.
 
-`mnemonicPhraseLanguage` is the language in which the mnemonic phrase is supplied. Acceptable values are
-"English", "Chinese", "Traditional Chinese", "French", "Italian", "Japanese", "Korean", and "Spanish".
+`consumingDerivationPathOpt`, if supplied, is the derivation path from the generated seed number to be used to generate
+the consuming wallet. By convention, it is "m/44'/60'/0'/0/0", but you can supply whatever path you want. Note that if
+you change any of the numbers ending in ', you may have trouble getting other software and hardware to work with your
+wallet. If you don't supply `consumingDerivationPathOpt`, you must supply `consumingPrivateKeyOpt`.
+
+`consumingPrivateKeyOpt`, if specified, is the private key of the consuming wallet, represented as a string of 64
+hexadecimal digits. This value supersedes `consumingDerivationPathOpt` if both are supplied; but if you don't supply 
+that value, you must supply this one.
 
-`consumingDerivationPath` is the derivation path from the mnemonic phrase that was used to generate the consuming
-wallet. By convention, it is "m/60'/44'/0'/0/0", but in this message it is required and no defaulting is performed
-by the Node.
+`earningDerivationPathOpt` is the derivation path from the generated seed number to be used to generate the earning
+wallet. By convention, it is "m/44'/60'/0'/0/1", but you can supply whatever path you want. Note that if
+you change any of the numbers ending in ', you may have trouble getting other software and hardware to work with your
+wallet. If you don't supply `earningDerivationPathOpt`, you must supply `earningAddressOpt`.
 
-`earningWallet` is either the derivation path from the mnemonic phrase that was used to generate the earning
-wallet, or--if the derivation path is unknown or the earning wallet is not related to the mnemonic phrase--the
-address of the earning wallet.
+`earningAddressOpt`, if specified, is the address of the consuming wallet, represented as "0x" followed by a string
```

Comment:
@dnwiebe I notice there needs to be a correction to the wording here, where it says:
"if specified, is the address of the consuming wallet..."

should be:

if specified, is the address of the earning wallet..."